package.json

```diff
@@ -18,7 +18,7 @@
     "bundle": "rollup -c build/rollup.config.ts",
     "bundle:min": "rollup -c build/rollup.config.min.ts",
     "build:dts": "vue-tsc --declaration --emitDeclarationOnly --skipLibCheck && npm run lintfix",
-    "prepare": "husky install",
+    "prepare": "husky install && npm run build",
```

Comment:
When you build while `prepare`, the `dist` directory is generated & as the CI checks show, the minified etc. versions will have lint issues. 

Thanks for opening the PR but I think having a `--watch` script would be more helpful than having `build` while `prepare`

Reply:
My usecase is also ability to use branches and forks in npm install. Do you know some other solution?

But for local dev `--watch` could be useful.